Readme: Add instructions to run CLI in local environment (#678)
bump-sh/cli: README.md

  ```

- Run the test suites

  ```shell
  npm run test
  npm run test-coverage # Run tests with coverage
  ```

> ### Use package in local environment
> 
> You can run the package by running file `run.js`:

Reviewer: ```suggestion
### Use package in local environment

You can run the package by executing the file `bin/run.js` locally:
```

  ```shell
  bin/run.js
  ```

For example to generate a preview:

  ```shell
  ./bin/run.js preview path/to/file.json
  > Your preview is visible at: https://bump.sh/preview/42
  ```

> Please note that even if CLI is run locally, requests are sent to Bump API for real.

Reviewer: ```suggestion
Please note that even if CLI is running locally, by default requests are sent to [Bump.sh API](https://developers.bump.sh/).
```

If you have a local version of Bump API, you can run CLI 100% in local environment,

> with env variable `BUMP_HOST`:

Reviewer: ```suggestion
by setting the environment variable `BUMP_HOST`:
```
